tests/unit/test_partial_rerun_merge_app.py

```diff
@@ -121,3 +121,44 @@ def partial_rerun_merge_app(event, api, asset_cache, rerun_asset_cache):
 
     partial_merge_app_response = scheduled_app(RAW_EVENT, context)[0]
     assert partial_merge_app_response == expected_response
+
+
+def test_event_parsing_not_failing_on_missing_field(context):
+    """
+    When field is missing we're still able to parse the event and invoke correct
+    function
+    """
+
+    @corva.scheduled(merge_events=True)
+    def scheduled_app(event, api, cache):
+        pytest.fail("Scheduled app was unexpectedly called!")
+
+    @corva.partial_rerun_merge
+    def partial_rerun_merge_app(event, api, asset_cache, rerun_asset_cache):
+        return True
+
+    event_with_missing_parameter: dict = deepcopy(RAW_EVENT)
```

Comment:
You can avoid using additional module by doing something like this
` event_with_missing_parameter = dict(RAW_EVENT)`

Reply:
You are right, for nested dicts we should use deepcopy.